### PR TITLE
Change language from left/right to relative position to SD-card.

### PR DESCRIPTION
**Pull request by JabLuszko**
Where is left, where is right?! ;)

### CODE REVIEW

**Comment by reedy** [comment]
> Where is left, where is right?! ;)

What if you turn the ATV upside down? Which is left and right then?

File: README_64bit.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ See optional network config [steps](#manual-network-configuration) if you need t
  - Plug FAT formatted USB drive with the mad_autoconf.txt file into appropriate USB port
    - On the TX9S use the USB port farthest away from network port
    - On A95X F1 use the right USB port
```
Comment (reedy) [inline README_64bit.md]: still right here

File: README_64bit.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ How to use Auto-Config with the 64-bit ROMs
 
 <sup>2</sup> Recommended USB ports for flashing (use other if first doesn't work):
  - On TX9S, flash with the USB port closest to the network port
- - On X96 Mini use left USB port (recommended on Discord)
+ - On X96 Mini use port farther away from SD-card slot.
  - On A95X F1 use right USB port (recommended on Discord)
```
Comment (reedy) [inline README_64bit.md]: And here

Comment (JabLuszko) [reply README_64bit.md]: If you have A95X F1 that you flashed lately and know which ports are which just submit the PR. I don't have access to that and changed only the x96 mini that I have in front of me atm.

Comment (dkmur) [reply README_64bit.md]: A95x for burning closest to power connection for autoconfig closest to SD

Comment (JabLuszko) [reply README_64bit.md]: Thanks @dkmur, updated.

**Pull request merged**